com.wps.Office.appdata.xml

```diff
@@ -0,0 +1,192 @@
+        <li>Support automatic filtering - color sorting</li>
+        <li>Monochrome printing is supported in page Settings</li>
+        <li>Support sharing workbook</li>
+        <li>Print the entire workbook under the print preview TAB</li>
+        <li>The digital format supports the locale</li>
+        <li>Support custom views</li>
+        <li>Support to read and play the animation of smartart child nodes</li>
+        <li>Support "font preview" function</li>
+        <li>Combine audio and background music entry, add audio tools TAB, and adjust media object toolbar synchronously</li>
+        <li>Support batch delete remarks function</li>
+        <li>In the comments pane of the speaker view, add the zoom button and adjust the text size</li>
+        <li>Support to insert media embedded into documents through the interface entrance</li>
+      </ul>
+    </release>
+    <release version="10.1.0.5707~a21" date="2017-06-15"/>
+  </releases>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">none</content_attribute>
+    <content_attribute id="violence-fantasy">none</content_attribute>
+    <content_attribute id="violence-realistic">none</content_attribute>
+    <content_attribute id="violence-bloodshed">none</content_attribute>
+    <content_attribute id="violence-sexual">none</content_attribute>
+    <content_attribute id="violence-desecration">none</content_attribute>
+    <content_attribute id="violence-slavery">none</content_attribute>
+    <content_attribute id="violence-worship">none</content_attribute>
+    <content_attribute id="drugs-alcohol">none</content_attribute>
+    <content_attribute id="drugs-narcotics">none</content_attribute>
+    <content_attribute id="drugs-tobacco">none</content_attribute>
+    <content_attribute id="sex-nudity">none</content_attribute>
+    <content_attribute id="sex-themes">none</content_attribute>
+    <content_attribute id="sex-homosexuality">none</content_attribute>
+    <content_attribute id="sex-prostitution">none</content_attribute>
+    <content_attribute id="sex-adultery">none</content_attribute>
+    <content_attribute id="sex-appearance">none</content_attribute>
+    <content_attribute id="language-profanity">none</content_attribute>
+    <content_attribute id="language-humor">none</content_attribute>
+    <content_attribute id="language-discrimination">none</content_attribute>
+    <content_attribute id="social-chat">none</content_attribute>
+    <content_attribute id="social-info">none</content_attribute>
+    <content_attribute id="social-audio">none</content_attribute>
+    <content_attribute id="social-location">none</content_attribute>
+    <content_attribute id="social-contacts">none</content_attribute>
+    <content_attribute id="money-purchasing">none</content_attribute>
+    <content_attribute id="money-gambling">none</content_attribute>
+  </content_rating>
+</component>
+<component type="desktop-application">
+  <id>com.wps.Office.et</id>
```

Comment:
appdata items other than `com.wps.Office` aren't exposed in app stores. Maybe they should be removed?

Reply:
No, we want this - it's the same for LibreOffice. There are plans to have all this exposed within app stores (certainly GNOME Software) and we need the metadata there first (same as OARS).

Reply:
Right, they are not different Flatpaks, but they are different launchables. Flatpak allows exporting any "sub-IDs" in the appstream, although it's gently broken in the LibreOffice Flatpak at the moment.